oak_restricted_kernel_api/src/logging.rs

```diff
@@ -0,0 +1,57 @@
+//
+// Copyright 2022 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+use core::fmt::Write;
+
+use crate::syscall::{fsync, write};
+
+struct Stderr {}
+
+impl Stderr {
+    const STDERR_FD: i32 = 2;
+
+    fn flush() {
+        fsync(Self::STDERR_FD).unwrap();
+    }
+}
+
+impl core::fmt::Write for Stderr {
```

Comment:
Should we also support stdout?

Reply:
We could, and it would be fairly trivial to do so. But nobody has asked for it yet.